x-pack/plugins/infra/public/components/asset_details/tabs/processes/processes_table.tsx

```diff
@@ -302,10 +304,25 @@ const columns: Array<{
   },
 ];
 
-const CodeLine = euiStyled.div`
-  font-family: ${(props) => props.theme.eui.euiCodeFontFamily};
-  font-size: ${(props) => props.theme.eui.euiFontSizeS};
-  white-space: pre;
-  overflow: hidden;
-  text-overflow: ellipsis;
-`;
+const CodeLine = ({ command }: { command: string }) => {
```

Comment:
euiStyled is deprecated. Created this new component in order to replace it.

Reply:
Thanks for changing those!